appengine/go11x/tasks/create_task.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START cloud_tasks_appengine_create_task]
+
+// Package tasks contains snippets for using Cloud Tasks with App Engine.
+package tasks
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	cloudtasks "cloud.google.com/go/cloudtasks/apiv2beta3"
+	taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2beta3"
+)
+
+// CreateTask creates a new task in your App Engine queue.
+func CreateTask(locationID, queueID, message string) (*taskspb.Task, error) {
```

Comment:
Does this need to be exported?

Reply:
Not really, I was thinking of this as a utility function, from that mindset exporting made sense.

Ack.